test/components/views/rooms/wysiwyg_composer/EditWysiwygComposer-test.tsx

```diff
@@ -186,6 +186,7 @@ describe('EditWysiwygComposer', () => {
     it('Should focus when receiving an Action.FocusEditMessageComposer action', async () => {
         // Given we don't have focus
         customRender();
+        screen.getByLabelText('Bold').focus();
```

Comment:
Is the test still testing the right thing?
I mean focusing bold should not focus the textbox for sure :D
As far as I get it, it should test that the text box is not focused after `customRender`.

Reply:
Yes, the purpose is to change the focus. So I check if the focus is not on the textbox and after I have the next steps of test

Reply:
Got it :heavy_check_mark: